### ISSUE TITLE
Bug: Flowtipe not able to interpret ::timestamp on query steps

### ISSUE THREAD

**Issue by oaulicino** [turbot/flowpipe#516]
While testing flowpipe, I used the following pipeline configuration:

pipeline "my_pipeline" {

```
  step "query" "inspector_open_findings" {
    connection_string = "postgres://steampipe@localhost:9193/steampipe"
    sql = <<-EOQ
      select
        finding_account_id,
        severity,
        exploit_available,
        fix_available,
        type,
        first_observed_at,
        last_observed_at,
        description
      from
        aws_inspector2_finding 
      where 
        status != 'CLOSED' and
        fix_available = 'YES' and
        first_observed_at::timestamp >= now()::timestamp - interval '1 day'
      order by case  
        when severity = 'CRITICAL' then 1
        when severity = 'HIGH' then 2
        when severity = 'MEDIUM' then 3
        when severity = 'LOW' then 4
        when severity = 'INFORMATIONAL' then 5
        else 6 end;
    EOQ
  }
  output "inspector_output" {
    value = step.query.inspector_open_findings.rows
  } 
```

Even though the query works well in steampipe, the same does not happen in flowpipe. I got the following error:

[my_pipeline] Internal Error: Bad Request: Query input must define args if the sql has placeholders
[my_pipeline] Failed 11ms

I troubleshooted the issue by removing the `::timestamp` from the code and it worked (even though the result was empty. I know the pipeline configuration is correct, as I was able to remove the whole where block and I was able to see the output normally:
![image](https://github.com/turbot/flowpipe/assets/31516700/0558f3e9-774e-4cda-9fa5-0c20dcbc0509)

PS: I also tried the net_http_request plugin in queries and the same applies, I believe it is related to the request_headers that may not be interpreted properly, but I can't be sure in this case.

**Comment by vhadianto** [comment]
Thanks for the bug report @oaulicino, confirmed that I could reproduce this. Having a look.

**Comment by vhadianto** [comment]
The failure is caused by the query string validation to detect arguments, i.e. `$1`, `?`, or `:name` (Oracle), in the query and check if the there are arguments supplied. 

I can fix the regular expression to catch Postgres casting operator (`::`) but in retrospect, this is difficult to cover all possible use cases especially in with complex SQL queries.

Decided to remove this validation.

**Comment by oaulicino** [comment]
@vhadianto I see that this was closed, however I am still seeing the same error. 
I uninstalled and reinstalled flowpipe just in case, but it seems like the issue persists. Is there something I need to do?

**Comment by vhadianto** [comment]
@oaulicino the issue has been closed, but the release was just built: 

https://github.com/turbot/flowpipe/releases/tag/v0.2.0-beta.202312200046

If you download this latest release it does include this commit: c3537c474957a349620c3a80aa284e369d04b5cf

Note that it's a `pre-release` so it's not published to our Homebrew Tap.

**Comment by vhadianto** [comment]
@oaulicino we were going to bundle this fix in `v0.2.0` but it's a little delayed so we've decided to push `v0.1.1` instead that contains this fix.

https://github.com/turbot/flowpipe/releases/tag/v0.1.1
